demos/test-app/src/index.tsx

```diff
@@ -347,11 +347,13 @@ init();
 
 whoamiBtn.addEventListener("click", async () => {
   const canisterId = Principal.fromText(readCanisterId());
+  let agent = new HttpAgent({
+    host: hostUrlEl.value,
+    identity: delegationIdentity,
+  });
+  await agent.fetchRootKey();
```

Comment:
Fetching the root key is now necessary because the agent now verifies the replica signature on queries too.